blaze/expr/arithmetic.py

```diff
@@ -135,11 +155,11 @@ def symbol(self):
 
     @property
     def dshape(self):
-        return DataShape(*(shape(self._child) + (self._dtype,)))
+        return DataShape(*(shape(self._operands[0]) + (self._dtype,)))
```

Comment:
same here, no need to change this code because you're defining `self._child` to be this